doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -222,6 +222,7 @@ Rotate Tool                          | {kbd}`E`
 Scale Tool                           | {kbd}`R`
 Camera Tool                          | {kbd}`T`
 Crop Window Tool                     | {kbd}`C`
+Cycle Tool Orientation               | {kbd}`O`
```

Comment:
Would this be better just under "Rotate Tool" on L221, so that it's more closely associated with the tools it affects? And/or would it be easier understood as "Cycle TransformTool Orientation"?

Reply:
That sounds a bit better, changed in https://github.com/GafferHQ/gaffer/pull/5207/commits/c6b8f793d94feea1526781c49fce9a56c172971c